changelog/4935.feature.rst

```diff
@@ -0,0 +1,15 @@
+Add option ``use_cls_token`` to all tokenizers. If it is set to ``True``, the token ``__CLS__`` will be added to
+the end of the list of tokens.
+
+Add option ``return_sequence`` to all featurizers. By default all featurizers return a matrix of size
+(1 x feature-dimension). If the option ``return_sequence`` is set to ``True``, the corresponding featurizer will return
+a matrix of size (token-length x feature-dimension).
+
+Split featurizers into sparse and dense featurizers.
```

Comment:
is this relevant for the end user?

Reply:
Not really, will remove it from the changelog entry.